src/features/surveys/l10n/messageIds.ts

```diff
@@ -174,7 +174,9 @@ export default makeMessages('feat.surveys', {
       'Something went wrong when submitting your answers. Please try again later.'
     ),
     policy: {
-      link: m('https://zetkin.org/privacy'),
+      link: m(
+        process.env.ZETKIN_PRIVACY_POLICY_LINK || 'https://zetkin.org/privacy'
+      ),
```

Comment:
I'm afraid we can't do this here, because the default text needs to be static.

Instead of putting the environment variable here, can  you put the same `||` logic wherever `feat.surveys.surveyForm.policy.link` is used?

Or maybe remove the translated message altogether and we can just require all instances to have this environment variable to function correctly?

Reply:
Yeah, having multiple places the variable is set without a real use case for it seems weird. If it doesn't change with the translation, then I say we put it only in environment. And we can still fall back on the original, right?

Reply:
I think we should fall back to the translation, like this (wherever the message is used today):

```tsx
const privacyUrl = process.env.ZETKIN_PRIVACY_POLICY_LINK || messages.surveyForm.policy.link();
```

Now there's another potential problem here which is that the `ZETKIN_PRIVACY_POLICY_LINK` variable will not be available on the frontend. This is solved elsewhere by explicitly adding the environment variable to the `Environment` that can then be accessed by `useEnv()`.

Reply:
The reason why I prefer using the translation as the fallback is that for some instances (e.g. the main Zetkin instance used by most organizations) it is still useful to be able to have different translated versions of the same privacy policy, and for that the messages is a good solution.

Reply:
Aah, okay, yes. Then I understand the priority here, I think. I'll make the change.